MatrixSDK/MXSession.m

```diff
@@ -946,6 +946,12 @@ - (void)resume:(void (^)(void))resumeDone
     [self handleBackgroundSyncCacheIfRequiredWithCompletion:^{
         [self _resume:resumeDone];
     }];
+    
+    [self.matrixRestClient maxUploadSize:^(NSInteger maxUploadSize) {
```

Comment:
I'm pretty sure this is in the wrong place. Would appreciate a suggestion of where it should really go 🙂

Reply:
Done!